### ISSUE TITLE
When an Opportunity is marked Completed, participants should receive an email

### ISSUE THREAD

**Issue by ultrasaurus** [18F/midas#580]
(Email sent once the Task Creator has marked the Opportunity complete)
```
To: Participant
CC: Task Creator 
Subject Line: [Task name] is complete — thank you!  

Hi [user’s name],

Congrats! You’ve successfully completed “[task name here].” Your profile has been updated to reflect your achievement. 

Thanks again,

The [_________] Team
```

Note: this depends on: https://github.com/18F/midas/issues/580

**Comment by LisaLNelson** [comment]
https://www.surveymonkey.com/s/openoppsfy15

**Comment by dhcole** [comment]
@ultrasaurus is this blocked by anything or can it move forward?

**Comment by dhcole** [comment]
Email copy with the survey:

```md
(Email sent once the Task Creator has marked the Opportunity complete)

To: Participant
CC: Task Creator 
Subject Line: [Task name] is complete — thank you!  

Hi [user’s name],

Congrats! You’ve successfully completed “[task name here].” Your profile has been updated to reflect your achievement.

We at Open Opportunities are always looking for ways to make our program better. Please take a moment to complete this quick survey — your feedback will help us continue to improve your experience.

Thanks again,

The Open Opportunities Team


```

**Comment by dhcole** [comment]
Duplicate of #819. Will track there.
